app/views/employers/employer_profiles/_employer_form.html.erb

```diff
@@ -120,7 +120,8 @@
             <h2> Office Locations </h2>
             <% @row = false %>
             <%= f.fields_for :office_locations, errors: {}, fieldset: false do |office_location,block| %>
-              <%= render partial: 'shared/office_location_fields', locals: {f: office_location} %>
+              <%= render partial: 'shared/office_location_fields', locals: {f: office_location, child_index: 
+              f.object.office_locations.length - 1} %>
```

Comment:
Whenever we add a new office location the object.office_locations.length will change right and we would be requiring child_index.